src/ml_board/frontend/Dashboard/src/components/dashboard/Dashboard.tsx

```diff
@@ -11,7 +11,7 @@ export default function Dashboard() {
   const rows = useAppSelector(selectAllRows);
   const colNames: string[] = [];
 
-  // TODO: get them from the Redux store
+  // TODO: get them from the Redux store better, otherwise this checks the first row ONLY!!!
```

Comment:
Do we need an issue for this?

Reply:
yes, but I didn't want to create new one for that as we already have #119 #150